tests/tfm/tfm_psa_test/testcase.yaml

```diff
@@ -16,39 +12,109 @@ tests:
     tags: tfm_lvl1
     extra_args: "CONFIG_TFM_PSA_TEST_PROTECTED_STORAGE=y CONFIG_TFM_ISOLATION_LEVEL=1"
     timeout: 120
+    platform_allow: >
+      nrf5340dk_nrf5340_cpuapp_ns
+      nrf9160dk_nrf9160_ns
+    integration_platforms:
+      - nrf5340dk_nrf5340_cpuapp_ns
+      - nrf9160dk_nrf9160_ns
+      - nrf54l15pdk_nrf54l15_cpuapp_ns
```

Comment:
Reading the doc (https://docs.zephyrproject.org/latest/develop/test/twister.html#test-cases) 

`platform_allow: <list of platforms>`
_Set of platforms that this test case should only be run for. Do not use this option to limit testing or building in CI due to time or resource constraints, this option should only be used if the test or sample can only be run on the allowed platform and nothing else._

`integration_platforms: <YML list of platforms/boards>`
_This option limits the scope to the listed platforms when twister is invoked with the --integration option. Use this instead of platform_allow if the goal is to limit scope due to timing or resource constraints._

`nrf54l15pdk_nrf54l15_cpuapp_ns` should be part of the platform_allow as well, maybe one of the testers could give a comment @magnev , @stephen-nordic

Reply:
fixed